srcpkgs/python3-qrcode/template

```diff
@@ -1,18 +1,20 @@
 # Template file for 'python3-qrcode'
 pkgname=python3-qrcode
-version=6.1
-revision=5
+version=7.3.1
+revision=1
 wrksrc="qrcode-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-six"
-checkdepends="python3-six"
+checkdepends="python3-six python3-Pillow"
 short_desc="QR Code image generator (Python3)"
 maintainer="Charles E. Lehner <cel@celehner.com>"
 license="BSD-3-Clause"
 homepage="https://github.com/lincolnloop/python-qrcode"
 distfiles="${PYPI_SITE}/q/qrcode/qrcode-${version}.tar.gz"
-checksum=505253854f607f2abf4d16092c61d4e9d511a3b4392e60bff957a68592b04369
+checksum=375a6ff240ca9bd41adc070428b5dfc1dcfbb0f2507f1ac848f6cded38956578
+# packages needed to run tests are not packaged in void
```

Comment:
could you list a few here so when/if they are added, we know tests can be enabled here?